packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -615,10 +613,6 @@ describe('Launchpad: Setup Project', () => {
       })
     })
 
-    beforeEach(() => {
-      fakeInstalledDeps()
```

Comment:
@elevatebart why did this exist/work previously but it doesn't now?

Reply:
We were intercepting things before and within the tests...